core/src/main/scala/kafka/metrics/ClientMetricsConfig.scala

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package kafka.metrics
```

Comment:
@AndrewJSchofield @hachikuji @sarat-kakarla @mjsax Does it make more sense to move this class (as java) in `org.apache.kafka.common.config.` or `org.apache.kafka.common.config.internals` rather than in `core`. I see all configuration classes are defined in `common` itself.

Reply:
Atleast the config def to be moved?

Reply:
Moved to java.